cmd/k8s-kata-manager/main.go

```diff
@@ -27,6 +27,9 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/NVIDIA/k8s-kata-manager/internal/runtime"
```

Comment:
These imports are local ones. They should be grouped with the other local imports.  See the 3rd paragraph of imports starting from L41

Reply:
We should be able to run `make goimports` to update the imports.

Also, @tariq1890 / @visheshtanksale could we enable the import checks in golangci-lint?

Reply:
> These imports are local ones. They should be grouped with the other local imports. See the 3rd paragraph of imports starting from L41

Fixed.

Reply:
> We should be able to run `make goimports` to update the imports.
> 
> Also, @tariq1890 / @visheshtanksale could we enable the import checks in golangci-lint?



> We should be able to run `make goimports` to update the imports.
> 
> Also, @tariq1890 / @visheshtanksale could we enable the import checks in golangci-lint?

I did use goimports not sure why it didnt put the imports in the right place.